go.mod

```diff
@@ -43,7 +43,7 @@ require (
 	code.cloudfoundry.org/rep v0.0.0-20200325195957-1404b978e31e // indirect
 	code.cloudfoundry.org/rfc5424 v0.0.0-20180905210152-236a6d29298a // indirect
 	code.cloudfoundry.org/tlsconfig v0.0.0-20200131000646-bbe0f8da39b3 // indirect
-	github.com/DataDog/agent-payload v4.67.0+incompatible
+	github.com/DataDog/agent-payload v4.68.1-0.20210423123626-9ac85cd19ccb+incompatible
```

Comment:
TODO: Update with correct version once https://github.com/DataDog/agent-payload/pull/96 is merged/released